app/components/election-urgent-live-filters.js

```diff
@@ -0,0 +1,52 @@
+'use-strict'
+
+// https://github.com/EnCiv/undebate-ssp/issues/21
+
+import cx from 'classnames'
+import React, { useState } from 'react'
+import { createUseStyles } from 'react-jss'
+
+import SvgElectionUrgentFilter from '../svgr/election-urgent-filter'
+import SvgElectionLiveFilter from '../svgr/election-live-filter'
+
+function ElectionUrgentLiveFilters(props) {
+    const { onDone, name = 'election-urgent-live-filters', style, className } = props
+    const classes = useStyles()
+
+    const filterType = e => {
+        onDone({ value: e.currentTarget.id, valid: true })
+    }
+
+    return (
+        <div>
```

Comment:
A couple of comments...changing it to an inline-block will cause the two circles to appear next to each other on the same line vs below each other...unless I'm missing something. Also my solution is based on flexbox...I chose flexbox to ensure it behaved a specific way. I  could set it to inlineFlex but that also causes the circles to appear on the same line.  
Are you available for a quick call so I can show you what I mean?

Reply:
I'll add the enhancement so have the filter will remain selected and it will reflect it's been selected by keeping it "hover" size (per my suggested enhancement in my first note above).

Reply:
Awaiting clarification...as noted I don't think displaying as inline-block will result in the right behavior (i.e., side-by-side vs vertically aligned...maybe there is some other attribute I need to set in addition to inline-block to force vertical alignment. Also not using flex introduces some undesirable behavior...circles don't grow from the center and push things down and to the side otherwise.

Reply:
Pending other input I changed the circle container to be "inline-flex" (equivalent to inline-block) and then gave the outermost div a fixed width (using em's ) to force vertical alignment of the filters...otherwise they would be adjacent to each other. This means that if some decides to change the width to something larger relative to the inner divs you could end up with the filters being adjacent to each other...maybe that's okay?  But the solution I used is the only one that I could think of given the request to make these "inline-block". If you have other solutions let me know.